var/spack/repos/builtin/packages/py-neuror/package.py

```diff
@@ -15,15 +15,18 @@ class PyNeuror(PythonPackage):
     url = "https://pypi.io/packages/source/n/neuror/NeuroR-1.2.3.tar.gz"
 
     version('develop', branch='master')
-    version('1.2.3', sha256='eb1c06748bf7706919cd78ab92bbb2420d5ac1dcaf54566c6f5aca403c63c215')
+    version('1.4.1', sha256='86ba939f57e2742abfceca82df966b79bfcdd5cc20bedd4c96c63e5275cf0e61')
+    version('1.3.0', sha256='c06147a7f4e976cdb6cf0f7c82388d3e7e5880a8fb3ded14f4c3e7ea3711876b')
 
     depends_on('py-setuptools', type=('build', 'run'))
 
     depends_on('py-click@7.0:', type='run')
     depends_on('py-matplotlib@2.2.3:', type='run')
-    depends_on('py-morph-tool@0.1.14:', type='run')
-    depends_on('py-morphio@2.1.1:', type='run')
-    depends_on('py-neurom@2.0:', type='run')
+    depends_on('py-morph-tool@2.8.0:', type='run', when='@1.4.0:')
+    depends_on('py-morph-tool@0.1.14:2.7.9', type='run', when='@:1.3.99')
```

Comment:
`py-morph-tool@0.1.14:2.7.9` is a mistake by me. I will fix it. Considering usage of `py-neuror`. I expect that a replacement for `py-morphology-repair-workflow` will be declared in spack soon. It will use this module for sure.